common/vision/lasr_vision_msgs/srv/PointingDirection.srv

```diff
@@ -0,0 +1,5 @@
+# Image to run inference on
+
+sensor_msgs/Image image_raw
+---
+string direction
```

Comment:
I think this would be better off being a custom message, e.g. something like `Direction.msg`:

```
uint8 NONE=0
uint8 LEFT=1
uint8 RIGHT=2
```

I'll open a separate issue to capture this in a future PR.

Reply:
#154.